<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Elf</class>
 <widget class="QWidget" name="Elf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>7</horstretch>
    <verstretch>7</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Elf</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>1258</width>
     <height>698</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>7</horstretch>
     <verstretch>7</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <widget class="QWidget" name="init">
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1280</width>
       <height>720</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="username">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>200</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>用户名</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>270</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QPushButton" name="login">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>340</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rgst">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>390</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exit">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>440</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="func">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>7</horstretch>
      <verstretch>7</verstretch>
     </sizepolicy>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1280</width>
       <height>720</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>7</horstretch>
       <verstretch>7</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="re_login">
      <property name="geometry">
       <rect>
        <x>1160</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>注销</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1280</width>
        <height>720</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QTableView" name="Online_user_list">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>380</y>
          <width>256</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
       <widget class="QTabWidget" name="Battle_elf_list">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>90</y>
          <width>321</width>
          <height>391</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>经验厂</string>
         </attribute>
         <widget class="QTableWidget" name="Exp_elf_list">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>60</y>
            <width>256</width>
            <height>192</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="Exp_battle_start">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>310</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>开始战斗</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>古罗马角斗场</string>
         </attribute>
         <widget class="QTableView" name="tableView_2">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>70</y>
            <width>256</width>
            <height>192</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="Battle_start">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>310</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>开始战斗</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>欢迎来到伊尔芙的世界</string>
        </property>
       </widget>
       <widget class="QLabel" name="display_username">
        <property name="geometry">
         <rect>
          <x>218</x>
          <y>18</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>610</y>
          <width>71</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>我的背包</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QScrollArea" name="Storehouse_ELF">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>360</y>
          <width>1130</width>
          <height>280</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1128</width>
           <height>278</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="SH_showelf">
        <property name="geometry">
         <rect>
          <x>88</x>
          <y>28</y>
          <width>371</width>
          <height>311</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>150</y>
          <width>69</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QFrame" name="SHE_info">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>40</y>
          <width>501</width>
          <height>291</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </widget>
     </widget>
     <zorder>tabWidget</zorder>
     <zorder>re_login</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="reminder">
   <property name="geometry">
    <rect>
     <x>448</x>
     <y>340</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
